### PR TITLE
mds: added manual deployment

### PR DESCRIPTION
**Pull request by Nefelim4ag**
Based on:
http://www.sebastien-han.fr/blog/2013/05/13/deploy-a-ceph-mds-server/
and
http://www.sebastien-han.fr/blog/2012/07/04/remove-a-mds-server-from-a-ceph-cluster/

Signed-off-by: Timofey Titovets <nefelim4ag@gmail.com>

### CODE REVIEW

File: doc/rados/deployment/ceph-deploy-mds.rst [hunk 1]
```diff
@@ -23,6 +23,35 @@ Once you deploy monitors and OSDs you may deploy the metadata server(s). ::
...
+
+	$ sudo mkdir /var/lib/ceph/mds/ceph-a
+	
+3. Create the authentication key (only if you use cephX) ::
+
+	$ sudo ceph auth get-or-create mds.a mds 'allow ' osd 'allow *' mon 'allow rwx' > /var/lib/ceph/mds/ceph-a/ceph.keyring
+
+4. Start mds daemon ::
+
+	 $ sudo service ceph start mds.0
```
Comment (jcsp) [inline doc/rados/deployment/ceph-deploy-mds.rst]: s/mds.0/mds.a/

File: doc/rados/deployment/ceph-deploy-mds.rst [hunk 2]
```diff
@@ -23,6 +23,35 @@ Once you deploy monitors and OSDs you may deploy the metadata server(s). ::
...
+	 
+5. Check ::
+	
+	$ ceph mds stat
+	e22: 1/1/1 up {0=a=up:active}
```
Comment (jcsp) [inline doc/rados/deployment/ceph-deploy-mds.rst]: The MDS won't appear here unless someone has already been through the "fs new"  (http://ceph.com/docs/master/cephfs/createfs/) part of the process.  That needs indicating here somehow

**Comment by jcsp** [comment]
This might be better placed in /install/manual-deployment/ -- I don't think we want to scatter the manual versions of things side-by-side with the simple versions everywhere

File: doc/rados/deployment/ceph-deploy-mds.rst [hunk 3]
```diff
@@ -44,3 +73,15 @@ Coming soon...
...
+========================
+
+1. Remove mds section in your config file ::
+
+	[mds.a]
+	host = {hostname}
+	
+2. Remove mds from cluster ::
+
+	$ sudo ceph mds rm mds.a
```
Comment (jcsp) [inline doc/rados/deployment/ceph-deploy-mds.rst]: These instructions are likely to not work as-is on a real system.  "mds rm" can only be run on inactive MDSs, so this needs to explain about stopping the daemon, then running "mds fail" on the gid.  Actually, at that point I'm not sure you need to run "mds rm" at all -- try it out?

**Comment by Nefelim4ag** [comment]
Okay, i will redo, fix and then resend  pull request

**Comment by jcsp** [comment]
@Nefelim4ag are you still working on this?

**Comment by Nefelim4ag** [comment]
Sorry, but already not

**Comment by jcsp** [comment]
@Nefelim4ag ping -- are you likely to come back to this or shall I close it?

**Pull request closed without merging**